.github/workflows/pytest-gpu.yml

```diff
@@ -61,7 +61,11 @@ jobs:
         if [ "${{ matrix.name }}" == 'pytest-gpu-omp' ]; then
             pytest --cov --cov-config=.coveragerc --cov-report=xml examples/seismic/viscoelastic/viscoelastic_example.py
         fi
-
+        if [ "${{ matrix.name }}" == 'pytest-gpu-acc' ]; then
```

Comment:
would put in another one as not pytest but just a run i.i
```
name: Run with mpi
if: matrix.name == 'pytest-gpu-acc'
run: |
  DEVITO_MPI=1 mpirun -n 2 python3 examples/seismic/elastic/elastic_example.py
            DEVITO_MPI=1 mpirun -n 2 python3 examples/seismic/viscoacoustic/viscoacoustic_example.py
            DEVITO_MPI=1 mpirun -n 2 python3 examples/seismic/viscoelastic/viscoelastic_example.py
```

Reply:
With everything in the if, I prefer it as a conditional step (as above) rather than a step with conditional in run.

Reply:
So you mean like:
```
    - name: Test with pytest
      run: |
        pgaccelinfo
        pytest --cov --cov-config=.coveragerc --cov-report=xml ${{ matrix.test_file }}
        pytest --cov --cov-config=.coveragerc --cov-report=xml examples/seismic/elastic/elastic_example.py
        if [ "${{ matrix.name }}" == 'pytest-gpu-omp' ]; then
            pytest --cov --cov-config=.coveragerc --cov-report=xml examples/seismic/viscoelastic/viscoelastic_example.py
        fi
        if [ "${{ matrix.name }}" == 'pytest-gpu-acc' ]; then
            DEVITO_MPI=1 mpirun -n 2 py.test examples/seismic/elastic/elastic_example.py
            DEVITO_MPI=1 mpirun -n 2 py.test examples/seismic/viscoacoustic/viscoacoustic_example.py
            DEVITO_MPI=1 mpirun -n 2 py.test examples/seismic/viscoelastic/viscoelastic_example.py
        fi
```

Reply:
No I mean what I wrote in the first comment where I gave the code example for it.

Reply:
I think fixed.